requirements.txt

```diff
@@ -36,6 +36,7 @@ pysolr==3.6.0
 python-dateutil==2.8.1
 python-editor==1.0.4      # via alembic
 python-magic==0.4.15
+python2-secrets==1.0.5    # This can be removed after the Python 2->3 migration
```

Comment:
Put this line into `requirements-py2.txt` instead - it is used for such "Python 2" only requirements, that will no longer be used after migration finished

Reply:
Yeah... I should have known better. Thanks for pointing it out!